### PR TITLE
Fix ProfilerOptions() documentation

### PR DESCRIPTION
**Pull request by jacob-buehler**
Fixed a bad link.

Issue #689:
The hyperlink in the below sentence in the [Profiler options](https://capitalone.github.io/DataProfiler/docs/0.8.1/html/profiler_example.html#Profiler-options) section of the Data Profiler documentation needs to link to better documentation for more in-depth options documentation.

### CODE REVIEW

File: examples/structured_profilers.ipynb [hunk 1]
```diff
@@ -245,7 +245,7 @@
     "\n",
     "Below, let's remove the histogram and increase the number of samples to the labeler component (1,000 samples). \n",
     "\n",
-    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler)."
+    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler/docs/0.10.2/html/profiler.html#profile-options)."
```
Comment (taylorfturner) [inline examples/structured_profilers.ipynb:248]: what happens when we update the package to a new version?

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: examples/structured_profilers.ipynb (hunk 1 above)
Comment (taylorfturner) [reply examples/structured_profilers.ipynb:248]: this is very brittle in my opinion

File: examples/unstructured_profilers.ipynb [hunk 2]
```diff
@@ -178,7 +178,7 @@
     "\n",
     "Below, let's remove the vocab count and set the stop words. \n",
     "\n",
-    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler)."
+    "Full list of options in the Profiler section of the [DataProfiler documentation](https://capitalone.github.io/DataProfiler/docs/0.10.2/html/profiler.html#profile-options)."
```
Comment (taylorfturner) [inline examples/unstructured_profilers.ipynb:181]: as above

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: examples/structured_profilers.ipynb (hunk 1 above)
Comment (tyfarnan) [reply examples/structured_profilers.ipynb:248]: just tested and still works with 0.10.1 @jacob-buehler @taylorfturner

Comment (tyfarnan) [reply examples/structured_profilers.ipynb:248]: ....ah I see what u mean, breaks as in, it'll go stale

Comment (taylorfturner) [reply examples/structured_profilers.ipynb:248]: exactly -- goes stale, yes. Not that it breaks in the sense it doesn't go that versions link, but should go to the most up to date / max(version) of the docs

**Comment by jacob-buehler** [comment]
updated in #1002

**Pull request closed without merging**